.ci/teardown.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+log_copy_dest="$1"
+
+runtime_log_filename="kata-runtime.log"
+runtime_log_path="${log_copy_dest}/${runtime_log_filename}"
+runtime_log_prefix="kata-runtime_"
+
+proxy_log_filename="kata-proxy.log"
+proxy_log_path="${log_copy_dest}/${proxy_log_filename}"
+proxy_log_prefix="kata-proxy_"
+
+shim_log_filename="kata-shim.log"
+shim_log_path="${log_copy_dest}/${shim_log_filename}"
+shim_log_prefix="kata-shim_"
+
+crio_log_filename="crio.log"
+crio_log_path="${log_copy_dest}/${crio_log_filename}"
+crio_log_prefix="crio_"
+
+docker_log_filename="docker.log"
+docker_log_path="${log_copy_dest}/${docker_log_filename}"
+docker_log_prefix="docker_"
+
+# Copy log files if a destination path is provided, otherwise simply
+# display them.
+if [ ${log_copy_dest} ]; then
+	# Create the log files
+	journalctl --no-pager -t kata-runtime-cc > "${runtime_log_path}"
```

Comment:
Strictly, this should be looking for the runtime name that's been configured for docker/k8s. But for simplicity, how about we create two runtime log files, one for `kata-runtime-cc` and another for `kata-runtime-runv`?

Reply:
Agree, lets create two log files now and once we move on with the https://github.com/kata-containers/runtime/issues/33, we can modify it.